server/index-node/src/schema.graphql

```diff
@@ -57,6 +59,21 @@ type EthereumIndexingStatus implements ChainIndexingStatus {
   lastHealthyBlock: Block
 }
 
+type EntityChanges {
+  updates: [EntityData!]!
+  deletes: [String!]!
+}
+
+type EntityData {
+  name: String!
```

Comment:
AFAICT, this is the entity type. It might be clearer to call this field `entity_type` then.

Reply:
Sure!